INSTALL_MACOS.md

```diff
@@ -1,15 +1,15 @@
-Instructions for installing Natron from sources on macOS
+Instructions for installing Natron from sources on MacOS
 =======================================================
 
 These are step-by-step instructions to compile Natron on macOS.
 
-OS X 10.6 (a.k.a. Snow Leopard) and newer are supported when building with MacPorts, and Homebrew can be used to compile it on the latest macOS (although a few Homebrew casks have to be downgraded).
+MacOS 10.6 (a.k.a. Snow Leopard) and newer are supported when building with MacPorts, and Homebrew can be used to compile it on the latest version of MacOS.
```

Comment:
OS X 10.6, not MacOS (check wikipedia).
The last version is macOS, not MacOS

Reply:
Resolved.